tests/operator/operator.go

```diff
@@ -981,7 +984,11 @@ var _ = Describe("[sig-operator]Operator", Serial, decorators.SigOperator, func(
 			if checks.HasAtLeastTwoNodes() {
 				vmis, err = generateMigratableVMIs(2)
 				Expect(err).ToNot(HaveOccurred())
-				vmis = createRunningVMIs(vmis)
+
```

Comment:
The commit message states that you want to test the cluster upgrade scenario with secondary networks in VM, but this test checks kubevirt re-creation, what added value do you see in testing the NAD here?
From what I see when we delete KV all the VMs are deleted as well.

Reply:
since this test uses generateMigratableVMIs, which adds the secondary interface (as was in the original test before it was [dropped](https://github.com/kubevirt/kubevirt/commit/13c81473d9ddb5114c781195b83655ae7799e436) by mistake)
it needs the NAD, and i created it just before creating the VM, due to this comment
https://github.com/kubevirt/kubevirt/pull/15394#discussion_r2280852682

are you sure that if one deletes KV it delete all the VMs ?
anyhow it is not relevant in this case, due to the above reason

follow-up might be to add the interface just to the tests requires it + create the NAD only there